internal/webhook/module.go

```diff
@@ -21,8 +21,10 @@ import (
 	"errors"
 	"fmt"
 	"regexp"
+	"strings"
 
 	"github.com/go-logr/logr"
+	multierror "github.com/hashicorp/go-multierror"
```

Comment:
I think we should use [`errors.Join`](https://pkg.go.dev/errors#Join) from the standard library instead.